x/bank/internal/keeper/keeper.go

```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/auth"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
These shouldn't use authtypes, but rather bank types (types.AccountKeeper).

Reply:
so add an account interface? `authtypes` is only used for `Account` and `VestingAccount`

Reply:
@colin-axner yes -- in `x/bank/internal/types/expected_keepers.go` implement the `Account` interface.